src/components/modules/VerticalNav/VerticalNav.tsx

```diff
@@ -71,6 +88,26 @@ export const VerticalNavItem = (props: IProps) => {
 		);
 	};
 
+	const renderTypeButton = (additionalTooltipClassName?: string | string[]) => {
+		return renderWrapper(
+			<button
+				className={styles.VerticalNavButton}
+				type="button"
+				aria-label={ariaLabel || tooltip || undefined}
+				aria-expanded={ariaExpanded}
+				aria-controls={ariaControls || undefined}
+				onClick={() => {
+					if (action) {
+						action();
+					}
```

Comment:
I think you could do away with the `action` prop and just grab `onClick` off of `props` (it exists in `IReactComponentProps`). And I think you wouldn't need the check, just:

```typescript
onClick={onClick}
```

Reply:
Ah, nice idea, thanks for suggesting. Done!